news/202010061300.misc

```diff
@@ -0,0 +1 @@
+Replace pipenv with tox
```

Comment:
I think we've done more than just replace pipenv with tox (generate docs, generate spdx). Should we have other news files, too?

Reply:
I have made the news file for each commit, explaining further changes.